src/main/java/org/folio/linked/data/client/SrsClient.java

```diff
@@ -11,4 +11,7 @@ public interface SrsClient {
 
   @GetMapping(value = "/records/{inventoryId}/formatted?idType=INSTANCE")
   ResponseEntity<Record> getFormattedSourceStorageInstanceRecordById(@PathVariable("inventoryId") String inventoryId);
+
+  @GetMapping(value = "/records/{srsId}/formatted?idType=SRS_RECORD")
+  ResponseEntity<Record> getFormattedSourceStorageRecordBySrsId(@PathVariable("srsId") String srsId);
```

Comment:
minor - we can remove 'formatted' from method name to make the method name little shorter. Also remove from the existing method `getFormattedSourceStorageInstanceRecordById`

Reply:
done.